### ISSUE TITLE
Put generated typings in a separate folder?

### ISSUE THREAD

**Issue by samuelneff** [Jimdo/typings-for-css-modules-loader#70]
Is there any option to have the generated `*.css.d.ts` files put in a specific folder and not alongside the css file? like `/src/typings/styles` ?

**Comment by JohnWeisz** [comment]
Yep, this would be great. Currently, it's necessary to `.gitignore` the generated definition files, as they are getting created in place with the rest of the source code.

**Comment by JohnWeisz** [comment]
In fact, it would be even better to have the option to dump all type definitions into a single file.
